prometheus_kafka_consumer_group_exporter/parsing.py

```diff
@@ -44,17 +44,18 @@ def parse_key(bytes):
             key_dict['group'], remaining_key = read_string(remaining_key)
 
         else:
-            logging.error('Can\'t parse __consumer_offsets topic message key with'
+            logging.warning('Can\'t parse __consumer_offsets topic message key with'
```

Comment:
Would need to fix the indent on the below lines (ditto for other logging changes)